src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements GetFigureArea{
+    private double base1;
+    private double base2;
+    private double height;
+
+    @Override
+    public String toString() {
+        return "Figure - IsoscelesTrapezoid: base1 " + this.base1
+                + ", base2 " + this.base2 + " ,height " + this.height
+                + " ,area " + this.getArea() + " color "
+                + this.color;

```

Comment:
The `color` field is accessed directly here. Consider using a getter method if encapsulation is a concern, especially if the `color` field is private in the `Figure` class.